File: blog-java/src/main/java/fun/aprilsxz/blog/domain/po/Article.java
package fun.aprilsxz.blog.domain.po;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import java.time.LocalDateTime;
import lombok.Data;

/**
 * 
 * @TableName article
 */
@TableName(value ="article")
@Data
public class Article implements Serializable {
    /**
     * 主键
     */
    @TableId
    private String articleId;

    /**
     * 文章标题
     */
    private String articleTitle;

    /**
     * 用户id
     */
    private String userId;

    /**
     * 文字标签
     */
    private String articleTags;

    /**
     * 分类id
     */
    private Integer categoryId;

    /**
     * 文字内容
     */
    private String articleContent;

    /**
     * 封面图片
     */
    private String articleCover;

    /**
     * 发布状态：0表示草稿，1表示已发布
     */
    private Integer status;

    /**
     * 创建时间
     */
    private LocalDateTime createTime;

    /**
     * 更新时间
     */
    private LocalDateTime updateTime;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;

}

File: blog-vue/src/entity/article.ts
export interface Article {
  articleId?: string
  articleTitle: string
  articleCover: string
  userId: string
  categoryId: string
  articleTags: string
  articleContent: string
  createTime?: string
  updateTime?: string
  articleStatus: number
}

export interface ArticleBrief {
  articleId: string,
  articleTitle: string,
  articleTags: string,
  createTime: string,
  updateTime: string,
  articleCover: string,

  articleStatus: number,

  nickname: string,

  categoryName: string,
}

export interface ArticleDetail {
  articleId: string,
  articleTitle: string,
  articleTags: string,
  createTime: string,
  updateTime: string,
  articleCover: string,

  articleContent: string,

  userId: string,
  nickname: string,

  categoryId: string
  categoryName: string,
}



